src/main/java/com/stories/sprintsclient/SprintsClient.java

```diff
@@ -0,0 +1,42 @@
+package com.stories.sprintsclient;
+
+import java.util.List;
+
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+import com.stories.domain.SprintDomain;
+
+@Component
+public class SprintsClient {
+
+	public boolean existsSprintById(String id) {
+		RestTemplate restTemplate = new RestTemplate();
```

Comment:
You should inject this RestTemplate object like a bean with @Autowired, you don´t need create a new instant of a object that currently exist.